lib/ansible/plugins/terminal/enos.py

```diff
@@ -76,7 +76,7 @@ def on_authorize(self, passwd=None):
             # Note: python-3.5 cannot combine u"" and r"" together.  Thus make
             # an r string and use to_text to ensure it's text
             # on both py2 and py3.
-            cmd[u'prompt'] = to_text(r"[\r\n]?password: $",
+            cmd[u'prompts] = to_text(r"[\r\n]?password: $",
```

Comment:
Is this right?
`grep -r "u'prompts'" lib/ansible` -> zero results
`grep -r "u'prompt'" lib/ansible` -> many results

Reply:
+1, @amuraleedhar pls check your command spec for whether `prompt` or `prompts` is the correct key.